packages/formation/js/accordion.js

```diff
@@ -16,60 +16,63 @@ const isElementInViewport = (
 };
 
 const loadAccordionHandler = () => {
-  const usaAccordion = [
-    ...document.getElementsByClassName('usa-accordion'),
-    ...document.getElementsByClassName('usa-accordion-bordered'),
-  ];
-  const usaAccordionContentElements = [
-    ...document.getElementsByClassName('usa-accordion-content'),
-  ];
+  document
```

Comment:
@rianfowler I used the query select but couldn't use your selector due to the situation when a button is nested in an element as how it is in mobile view on vets. The sibling combinator wouldn't work.  I couldn't find a selector that would work with nested children, so this is what I came up with. Any suggestions are welcomed.